examples/docker-compose.yml

```diff
@@ -7,6 +7,12 @@ services:
       EULA: "TRUE"
       GAMEMODE: survival
       DIFFICULTY: normal
+    healthcheck:
+      test: CMD /usr/local/bin/mc-monitor status-bedrock --host 127.0.0.1 --port 19132 
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 60s
     ports:
```

Comment:
What do you think about just baking this into the Dockerfile like I do in the Java edition? Then `docker run` and compose usage get it automatically. 
https://github.com/itzg/docker-minecraft-server/blob/master/Dockerfile#L20

Reply:
Yeah, I had considered that - just wasn't sure because it's not supported by k8s.  But, it does make sense to help the docker deployments.  I'll make the change.